streaming-binlog/src/test/java/org/openmrs/analytics/FhirConverterTest.java

```diff
@@ -92,6 +108,27 @@ public void shouldFetchFhirResourceAndUploadToFhirStore() throws IOException {
 		Mockito.verify(parquetUtil, Mockito.never()).write(Mockito.<Resource> any());
 	}
 	
+	@Test
+	public void shouldGetUuidFromParent() throws SQLException, PropertyVetoException, ClassNotFoundException {
+		
+		Map<String, String> messageBody = DebeziumTestUtil.genExpectedBody();
+		Map<String, Object> messageHeaders = DebeziumTestUtil.genExpectedHeaders(Operation.CREATE, "patient");
+		statement = jdbcConnectionUtil.createStatement();
+		String uuid = getUuidUtil.getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString());
+		
+		lenient().when(jdbcConnectionUtil.createStatement()).thenReturn(statement);
+		
+		lenient().when(getUuidUtil.getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString())).thenReturn(uuid);
+		
+		// Actual event that will trigger process().
+		eventsProducer.sendBodyAndHeaders(messageBody, messageHeaders);
+		
+		Mockito.verify(jdbcConnectionUtil, times(1)).createStatement();
+		Mockito.verify(jdbcConnectionUtil).createStatement();
```

Comment:
What are these two verifications doing? Aren't these simply because of the call you have on line 116 and not really anything called during `eventProducer.sedBodyAndHeaders`?